airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -88,6 +88,9 @@ def parse_response(self, response: requests.Response, stream_state: Mapping[str,
         else:
             yield from data
 
+    def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
```

Comment:
From my understanding, the purpose of creating a separate `get_records_field` function is to enable calling this logic without the stream state argument?

Reply:
```suggestion
    def parse_response(self, response: requests.Response, *args, **kwargs) -> Iterable[Mapping]:
```
I'd rather keep the logic in the same function but change its signature to allow to call it with the single response argument.

Reply:
Yeah, you're right, but I changed my approach in [83afd35](https://github.com/airbytehq/airbyte/pull/12374/commits/83afd357acbeb88d9333ea79ca1a38310bdf3d53).